src/Exercism.Representers.CSharp/Normalization/NormalizeDictionaryInitialization.cs

```diff
@@ -0,0 +1,183 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Runtime.InteropServices.ComTypes;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Operations;
+using Serilog;
+
+namespace Exercism.Representers.CSharp.Normalization
+{
+    public class NormalizeDictionaryInitialization : CSharpSyntaxRewriter
+    {
+        private class InvalidKindException : Exception
+        {
+            public SyntaxKind Kind;
+        }
+
+        private readonly SemanticModel semanticModel;
+
+        public NormalizeDictionaryInitialization(SemanticModel semanticModel)
+        {
+            if (semanticModel == default)
+            {
+                Log.Error(
+                    $"{nameof(NormalizeDictionaryInitialization)}: semantic model is null - this normalization will fail");
+            }
+
+            this.semanticModel = semanticModel;
+        }
+
+        public override SyntaxNode VisitInitializerExpression(InitializerExpressionSyntax node)
```

Comment:
Maybe you could split this method into several smaller helper methods? I found it a bit tricky to read :)

Reply:
semantic model will not be used